### PR TITLE
[FLINK-18264][doc-zh] Translate External Resources page to Chinese

### PR DESCRIPTION
**Pull request by KarmaGYZ**
## What is the purpose of the change

Translate External Resources page to Chinese

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

**Comment by KarmaGYZ** [comment]
cc @xintongsong

File: docs/ops/external_resources.zh.md [hunk 1]
```diff
@@ -234,127 +230,122 @@ class FPGAInfo extends ExternalResourceInfo {
 </div>
 </div>
 
-Create a file with name `org.apache.flink.api.common.externalresource.ExternalResourceDriverFactory` in `META-INF/services/`
-and write the factory class name (e.g. `your.domain.FPGADriverFactory`) to it.
+在 `META-INF/services/` 中创建名为 `org.apache.flink.api.common.externalresource.ExternalResourceDriverFactory` 的文件，向其中
+写入工厂类名，如 `your.domain.FPGADriverFactory`。
 
-Then, create a jar which includes `FPGADriver`, `FPGADriverFactory`, `META-INF/services/` and all the external dependencies.
-Make a directory in `plugins/` of your Flink distribution with an arbitrary name, e.g. "fpga", and put the jar into this directory.
```
Comment (xintongsong) [inline docs/ops/external_resources.zh.md:241]: Does the created directory need to have the same name as resource name configured in `external-resources`?

File: docs/ops/external_resources.zh.md [hunk 2]
```diff
@@ -234,127 +230,122 @@ class FPGAInfo extends ExternalResourceInfo {
...
 
-We provide a first-party plugin for GPU resources. The plugin leverages a discovery script to discover indexes of GPU devices, which can
-be accessed from the resource *information* via the property "index". We provide a default discovery script that can be used to discover
-NVIDIA GPUs. You can also provide your custom script.
+我们为 GPU 提供了第一方插件。该插件利用一个脚本来发现 GPU 设备的索引，该索引可通过“index”从 `ExternalResourceInfo` 中获取。我们提供了一个
+默认脚本，可以用来发现 NVIDIA GPU。您还可以提供自定义脚本。
 
-We provide [an example](https://github.com/apache/flink/blob/master/flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/gpu/MatrixVectorMul.java)
-which shows how to use the GPUs to do matrix-vector multiplication in Flink.
+我们提供了[一个实例程序](https://github.com/apache/flink/blob/master/flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/gpu/MatrixVectorMul.java)，
```
Comment (xintongsong) [inline docs/ops/external_resources.zh.md:254]: 实例 -> 示例

File: docs/ops/external_resources.zh.md [hunk 3]
```diff
@@ -234,127 +230,122 @@ class FPGAInfo extends ExternalResourceInfo {
...
+我们为 NVIDIA GPU 提供了一个默认脚本，位于 Flink 发行版的 `plugins/external-resource-gpu/nvidia-gpu-discovery.sh`。该脚本通过 `nvidia-smi` 工具获取
+当前可见 GPU 的索引。它尝试返回一个 GPU 索引列表，其大小由 `external-resource.<resource_name>.amount` 指定，如果 GPU 数量不足，则以非零退出。
 
-For standalone mode, multiple TaskManagers might be co-located on the same machine, and each GPU device is visible to all
-the TaskManagers. The default discovery script supports a coordination mode, in which it leverages a coordination file to
-synchronize the allocation state of GPU devices and ensure each GPU device can only be used by one TaskManager process. The relevant arguments are:
+在 Standalone 模式中，多个 TaskManager 可能位于同一台机器上，并且每个 GPU 设备对所有 TaskManager 都是可见的。默认脚本提供 GPU 协调模式，
+在这种模式下，脚本利用文件来同步 GPU 的分配情况，并确保每个GPU设备只能由一个TaskManager进程使用。相关参数为：
 
-  - `--enable-coordination-mode`: Enable the coordination mode.
```
Comment (xintongsong) [inline docs/ops/external_resources.zh.md:341]: By default is this enabled or disabled?

File: docs/ops/external_resources.zh.md [hunk 4]
```diff
@@ -23,98 +23,90 @@ specific language governing permissions and limitations
...
-  value of `external-resource.<resource_name>.amount`.
+首先，你需要使用分隔符“;”将所有使用的扩展资源类型的资源名称添加到 **扩展资源列表（配置键“external-resources”）** 中，例如，“external-resources: gpu;fpga”定义了两个扩展资源“gpu”和“fpga”。
+只有此处定义了扩展资源名称（**\<resource_name\>**），相应的资源才会在扩展资源框架中生效。
 
-  - **Config key in Kubernetes** (`external-resource.<resource_name>.kubernetes.config-key`): *optional*. If configured,
-  external resource framework will add `resources.limits.<config-key>` and `resources.requests.<config-key>` to the main
-  container spec of TaskManager and set the value to the value of `external-resource.<resource_name>.amount`.
+对于每个扩展资源，有以下配置选项。下面的所有配置选项中的 **\<resource_name\>** 对应于 **扩展资源列表** 中列出的名称：
 
-  - **Driver Factory** (`external-resource.<resource_name>.driver-factory.class`): *optional*. Defines the factory class
```
[the remark this answers is not in the document]
Comment (KarmaGYZ) [reply docs/ops/external_resources.zh.md:87]: The operator could not get any information of the external resource from `RuntimeContext`. But the requested external resource will still exist in the `TaskManager` container.

File: docs/ops/external_resources.zh.md (hunk 1 above)
Comment (KarmaGYZ) [reply docs/ops/external_resources.zh.md:241]: No, the name of the directory has no influence on the Plugin Mechanism.

File: docs/ops/external_resources.zh.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (xintongsong) [reply docs/ops/external_resources.zh.md:87]: Ok, that makes sense. I would suggest to explain this for both languages.

**Comment by xintongsong** [comment]
@flinkbot run azure

**Pull request closed without merging**